tests/e2e_tests.py

```diff
@@ -39,20 +46,28 @@ def test_explicit_paths(link_localtime_mock, get_timezone_for_ip_mock):
             FAKE_TIMEZONE, zoneinfo_path, localtime_path,
         ),
     )
+    assert_true(
+        export_etc_timezone_mock.called_once_with(
+            FAKE_TIMEZONE, etc_timezone_path,
```

Comment:
No, just "tox".  It says it ran 19 tests for each of 2.7, 3.4, 3.5.  That suggests that it is in fact running everything in both files.

I think I've figured it out.  The issue is why it is raising PermissionErrors within Travis but not locally.  I think it is because I am on OSX, which does not have the /etc/timezone file, so the function is getting called but not reaching the point where it would hit the PermissionError.